packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Examples.scss

```diff
@@ -6,17 +6,29 @@
       margin: 0 0 15px 0;
       max-width: $maxFieldWidth;
     }
+
+    .ms-TextFieldStandaloneLabel {
+      margin: 0 0 7px 0;
```

Comment:
Since this is new styling, can you do it with inline styles or mergeStyles instead?